Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -800,9 +813,14 @@ internal void AfterUpdateScene()
             }
 #endif
 
+#if VERBOSE_LOGS
+            Debug.Log($"Scene updated.");
+#endif
```

Comment:
Do we need this? Might be good we log messages with the "[MeshSync]" prefix, so users are aware where these are coming from

Reply:
I think it's good to have, it's very difficult to debug without them.
Users should never see this because of the #define but I've added "[MeshSync]".